net/quassel-irssi/Makefile

```diff
@@ -0,0 +1,47 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=quassel-irssi
+
+# quassel-irssi upstream doesn't release versions (at least, at the moment),
+# so use 0 here (in case they decide to later) for now.
+# I'm not sure if there's an openwrt convention here.
+PKG_VERSION:=0
+PKG_SOURCE_VERSION:=cbd9bd7f4ac44260d9fcafb809fdf153cde193e5
+PKG_RELEASE:=1.$(PKG_SOURCE_VERSION)
+
+PKG_LICENSE:=GPL-3.0+
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/phhusson/quassel-irssi
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+
+PKG_MAINTAINER:=Ben Rosser <rosser.bjr@gmail.com>
+
+PKG_BUILD_DEPENDS:=irssi quasselc glib2
+PKG_INSTALL:=1
```

Comment:
Three notes on this file

 - Copyright header is missing
 - `PKG_VERSION` and its friends need to be revised the same way as was said above about quasselc
 - I guess `PKG_BUILD_DEPENDS` can also be dropped here